app/views/taggings/_form_for_topics.html.erb

```diff
@@ -1,16 +1,15 @@
-<h3>Topics</h3>
+<h3>Specialist topic pages</h3>
 
 <%= f.input :topics,
     collection: linkables.topics, as: :grouped_select, group_method: :last,
-    placeholder: "Choose Topics...",
+    placeholder: "Choose Specialist topic pages...",
```

Comment:
I'm sure this is totally fine, but just flagging that there's no note that this placeholder text should be updated in Jen's document. https://docs.google.com/spreadsheets/d/1OLw4B46ZqTgsuo762Kjxb-M6_rfX_dW23qp8bNhZrMo/edit#gid=994922232 Maybe just triple check!